sdk-cpp/src/internal/com/do_download_property_internal.cpp

```diff
@@ -109,11 +109,7 @@ CDownloadPropertyValueInternal::CDownloadPropertyValueInternal(const CDownloadPr
 #endif
```

Comment:
related to my question earlier -- is the change now to not throw exceptions at all, or just to not surface exceptions?  I'm guessing it's the latter, but the previous implementation of this method was:

* Exceptions enabled: throw bad_alloc
* Exceptions disabled: terminate process

Now the behavior is (implied) Exceptions disabled: throw bad_alloc